src/Symfony/Component/Validator/Tests/Constraints/UniqueValidatorTest.php

```diff
@@ -16,6 +16,14 @@
 use Symfony\Component\Validator\Exception\UnexpectedValueException;
 use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
 
+class CallableClass
```

Comment:
Please declare the test case class first. This fixture class should be moved to the end of the file.